package.json

```diff
@@ -146,6 +146,7 @@
     "@types/tar-fs": "2.0.1",
     "@types/uuid": "9.0.0",
     "@types/ws": "8.5.4",
+    "buffer": "^5.7.1",
```

Comment:
@goosewobbler think you got an extra dep in here

Reply:
damn buffer again